orb/commands/@common.yml

```diff
@@ -117,6 +117,15 @@ docker-login:
         command: |
           silta ci image login
 
+docker-setup:
+  description: "Set up Docker."
+  steps:
+    - setup_remote_docker:
+        # Note: default Docker version is 17.09.
+        # Using BuildKit requires version 18.09 and dockerignore per dockerfile from 19.03
+        # Alpine 3.14+ builds require 20.10.0+
+        version: 20.10.7
```

Comment:
this is probably out of scope for this change, but I think the version configuration is obsolete / redundant at this point. Need to confirm, but i believe the default version is 24 now, so all requirements are available by default.
https://circleci.com/docs/remote-docker-images-support-policy/#image-lifespan-eol

Reply:
Good point, I removed the version key. Also added another commit to keep backwards compability for projects still using the non-split jobs.